frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts

```diff
@@ -1031,6 +1065,43 @@ export class ChallengesettingsComponent implements OnInit, OnDestroy {
     }
   }
 
+  /**
+   * Edit leaderboard description of the challenge
+   */
+  editLeaderboardDescription() {
+    const SELF = this;
+    SELF.apiCall = (params) => {
+      const BODY = JSON.stringify(params);
+      SELF.apiService
+        .patchUrl(SELF.endpointsService.editChallengeDetailsURL(SELF.challenge.creator.id, SELF.challenge.id), BODY)
+        .subscribe(
+          (data) => {
+            SELF.challenge.leaderboard_description = data.leaderboard_description;
+            SELF.globalService.showToast('success', 'The leaderboard description is successfully updated!', 5);
```

Comment:
@gautamjajoo please change the message to `Leaderboard description updated successfully!`

Reply:
Done